uploadNeuroDB/HRRT_PET_insertion.pl

```diff
@@ -580,7 +580,7 @@ sub run_hrrt_archival {
 
     my $tar_cmd = "tar -C $to_tar_dir -czf $final_target $study_dir";
     print "\nCreating a tar with the following command: \n $tar_cmd\n" if $verbose;
-    system($tar_cmd);
+    #system($tar_cmd);
```

Comment:
I guess you commented that for testing purposes?

Reply:
Ahah you are too fast, I am reverting ! This shouldn't be there, you are right.